src/components/common/AccountSkipThisStep.js

```diff
@@ -1,30 +1,47 @@
 import React from 'react'
-import { Segment } from 'semantic-ui-react'
+import styled from 'styled-components'
+import DocImg from '../../images/icon-doc.svg'
 
-import FormButton from '../common/FormButton'
+const SetUpPhraseBlock = styled.div`
+   padding: 20px 0 0 0;
+   border-top: 4px solid #f8f8f8 !important;
+   margin-top: 48px;
 
-import styled from 'styled-components'
+   .headline {
+      font-weight: 600;
+      color: #24272a;
+   }
+
+   .desc {
+      color: #4a4f54;
+      font-size: 11px;
+   }
 
-const CustomSegment = styled(Segment)`
-   &&& {
-      padding: 20px 0 0 0;
+   .button {
+      background: left 10px center no-repeat url(${DocImg});
+      background-size: 20px;
+      border: 2px solid #e6e6e6;
+      background-color: #f8f8f8;
+      border-radius: 3px;
       color: #24272a;
-      border-top: 4px solid #f8f8f8 !important;
-      margin-top: 48px;
-      line-height: 24px;
+      display: inline-block;
+      cursor: pointer;
+      padding: 10px 10px 10px 40px;
+      margin-top: 10px;
+      span {
+         font-weight: 600;
+      }
    }
 `
 
 const AccountSkipThisStep = ({ skipRecoverySetup }) => (
-   <CustomSegment basic>
-      <span className='font-bold'>Skip this step</span> if you want to backup your account keys manually. However, skipping this will make it impossible for us to assist you with account recovery in the future. 
-      <FormButton
-         onClick={skipRecoverySetup}
-         color='link'
-      >
-         I choose to backup my account manually
-      </FormButton>
-   </CustomSegment>
+   <SetUpPhraseBlock onClick={skipRecoverySetup}>
+      <div className='headline'>Switch to Manual Backup</div>
+      <div className='desc'>With this option, NEAR will not be able to assist you with account recovery.</div>
```

Comment:
NEAR won't be able to assist with any option to be fair

Reply:
Yes, suggest we change copy to "With this option, only your seed phrase will help with account recovery." @kcole16 any thoughts on the copy?

Reply:
How about "With this option, you must write down and save a 12 word phrase to recover your account."